boards/arm/lpcxpresso55s69/CMakeLists.txt

```diff
@@ -33,4 +33,65 @@ if (CONFIG_BUILD_WITH_TFM)
 	if(${SREC_CAT} STREQUAL SREC_CAT-NOTFOUND)
 		message(FATAL_ERROR "'srec_cat' not found. Please install it, or add it to $PATH.")
 	endif()
+
+	if (CONFIG_TFM_BL2_FALSE)
+		#merge tfm_s and zephyr_ns to a single image
+		set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+			COMMAND ${SREC_CAT}
+			ARGS ${CMAKE_BINARY_DIR}/tfm/install/outputs/LPC55S69/tfm_s.bin -Binary
+			${CMAKE_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME} -Binary
+				-offset 0x40000 #needs to be the same value as image-1 partion
+				-o ${CMAKE_BINARY_DIR}/tfm_image.bin -Binary
+		)
+	else()
```

Comment:
This will be executed if `CONFIG_TFM_BL2_TRUE=y` but also if `CONFIG_TFM_BL2_DEFAULT=y` and the TFM config file has `TFM_BL2=False`.

I expect things will break in that case.